src/query/xasl_cache.c

```diff
@@ -185,6 +186,9 @@ static LF_ENTRY_DESCRIPTOR xcache_Entry_descriptor = {
 #define XCACHE_RT_TIMEDIFF_IN_SEC	360	/* 10 minutes */
 #define XCACHE_RT_MAX_THRESHOLD		10000	/* 10k pages */
 #define XCACHE_RT_FACTOR		10	/* 10x or 0.1x cardinal change */
+#define XCACHE_RT_CLASS_STAT_NEED_UPDATE(class_pages,heap_pages) \
```

Comment:
sorry I don't really understand the condition. 
What does it mean and why we need it?